lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -3,7 +3,17 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
   This module handles Req/Resp domain requests.
   """
   require Logger
-  alias LambdaEthereumConsensus.Libp2pPort
+  alias LambdaEthereumConsensus.{Libp2pPort, P2P}
+  alias LambdaEthereumConsensus.Store.BlockStore
+
+  # This is the `ForkDigest` for mainnet in the capella fork
+  # TODO: compute this at runtime
+  @fork_context "BBA4DA96" |> Base.decode16!()
+
+  # This is the `Resource Available` error message in binary
+  # TODO: compute this and other messages at runtime
+  @error_message_resource_available "Resource Unavailable"
```

Comment:
nit: `error_message_resource_unavailable`

Reply:
Also, the comment mentions "Resource Available" when it should be "Resource Unavailable"